ui/app/styles/core/helpers.scss

```diff
@@ -165,13 +165,13 @@
   text-transform: lowercase;
 }
 .has-bottom-margin-xs {
-  margin-bottom: $spacing-xs;
+  margin-bottom: $spacing-xs !important;
```

Comment:
Are these required because `chart-title` or `stat-title` also have a margin-bottom value? Could we not update the other classes to not include margin values instead of adding `!important` (which should be used sparingly), since this helper is far-reaching and the charts and stats are only used in client counts so far.

Reply:
Yeah, I went back and forth on this. I can just add them to chart-title and stat-title, but sometimes they are overridden by the title class. Let me see if I can approach another way.